servicetest/filesystem/test_filesystem.py

```diff
@@ -202,3 +203,56 @@ def test_bindmount_sockets(self, flag):
 
         if os.path.exists(absoluteTestFile):
             os.remove(absoluteTestFile)
+
+    @pytest.mark.xfail("platform.release() <= \"3.18.0\"")
+    @pytest.mark.parametrize("flag", [True, False])
+    def test_tmpfs_writebuffer_size(self, flag):
+        """ The SoftwareContainer will create a tmpfs in the location of the
+        containers temporary directory location if the write buffer is enabled.
+        Test that the tmpfs mounted in the containers temporary directory
+        structure works as expected.
+
+        Note: The filesystem is hard coded to 100MB at the time this test was
+        written, but will be configurable via the DATA field in a short while
+        """
+
+        absoluteTestFile = os.path.join("/tmp/container/SC-0/rootfs-upper", TESTFILE)
+
+        ca = Container()
+
+        if flag is True:
```

Comment:
I'm not a fan of using parametrization like this. What's the benefit compared to having two tests? The code is mostly not reused.